steps/executexcuitestonbrowserstack/0.0.1/step.yml

```diff
@@ -0,0 +1,79 @@
+title: executeXCUITestOnBrowserstack
+summary: |
+  Execute XCUITests on Browserstack Platform with retry logic
+description: |
+  This step will help you to execute ios Automation Tests XCUITests on Browserstack platform. Automatically manages device queue and retries failed tests based on the configured value also reports test status.
+website: https://github.com/satheeshkumarf/bitrise-step-executexcuitestonbrowserstack
+source_code_url: https://github.com/satheeshkumarf/bitrise-step-executexcuitestonbrowserstack
+support_url: https://github.com/satheeshkumarf/bitrise-step-executexcuitestonbrowserstack/issues
+published_at: 2019-08-18T11:45:04.85055+04:00
+source:
+  git: https://github.com/satheeshkumarf/bitrise-step-executexcuitestonbrowserstack.git
+  commit: e55e1d9e0aa113c2f32ae2cec01317894b58a75f
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- macos
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: browserstack.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- BROWSERSTACK_APP_URL: null
+  opts:
+    is_expand: true
+    is_required: true
+    summary: IPA URL of the APP uploaded to Browserstack Server. You can use browserstack-upload
+      bitrise step to upload your ipa.
+    title: Browserstack IPA URL
+- TESTS_FIILE_PATH: null
+  opts:
+    description: |
+      Sample File format:
+          SearchListingsTest/testXYZ=iPhone 6-11.2
+          SearchFilterTest/testABC=iPhone XS-12.1
+    is_expand: true
+    is_required: true
+    summary: Specifiy the path to file name which contains your test and device details.
+      Use .txt file to specify your test name and device details separated by equal
+      to sign.
+    title: Test File Path
+- BROWSERSTACK_XCTEST_URL: null
+  opts:
+    is_expand: true
+    is_required: true
+    summary: Build your XCUITEST and publish it to Browserstack; Specify the test_url
+      path returned from browserstack in this variable
+    title: Specify the XCUITest url from browserstack
+- BROWSERSTACK_USERNAME: null
+  opts:
+    is_expand: true
+    is_required: true
+    title: Browserstack Username
+- BROWSERSTACK_PASSWORD: null
```

Comment:
please use https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#secret-environment-variables-in-steps secrets for this kind of inputs